docs/UserGuide.md

```diff
@@ -577,7 +589,7 @@ Example:
 
 :exclamation: **Caution:**
 * NurseyBook will not automatically refresh the displayed task list to reflect instantaneous changes, such as overdue tasks and new dates of recurring tasks.
-* However, you can manually trigger this refresh, and one way is to enter [`viewTasks`](#view-all-tasks-viewtasks) in the command box.
+* However, you can manually trigger this refresh, and one way is to enter [`viewTasks`](#view-all-tasks-viewtasks) in the command box. Entering task-related command would trigger this refresh.
```

Comment:
Maybe it would be better to use either `Entering a task-related command` or `Entering task related commands`?